src/modules/commands/discord/urbandictionary.ts

```diff
@@ -0,0 +1,83 @@
+import { Command } from '@commands/CommandHandler'
+import got from 'got'
+import { isRecord } from '@app/validators/object'
+import camelcaseKeys from 'camelcase-keys'
+import discord from 'discord.js'
+
+type definition = {
+  definition: string
+  permalink: string
+  thumbsUp: number
+  soundUrls: string[]
+  author: string
+  word: string
+  defid: number
+  currentVote: string
+  writtenOn: string
+  example: string
+  thumbsDown: number
+}
+
+export default new Command<discord.Message>({
```

Comment:
`commands/discord` are meant for commands that use Discord specific features and need the `raw` parameter set to `discord.Message`. However you do not use `raw` here, so you should move it to `commands/common`.

Reply:
I want to make it into an embed later, should I still move it?

Reply:
All good then.